package syncro.pb;

option java_package = "uk.me.grambo.syncro.pb";
option optimize_for = LITE_RUNTIME;

message AdminParameter {
	required string name = 1;
	optional string string_value = 2;
	optional int32 int_value = 3;
};

message GenericAdminCommand {
	optional string command = 1;
	repeated AdminParameter params = 3;
};

message AdminAck {
	optional bool ok = 1;
	optional int32 error_code = 2;
};